styles.css

```diff
@@ -51,14 +43,22 @@ section {
   padding: 0 0.5rem;
   border: 1px solid rgba(0, 0, 0, 20%);
   border-radius: 0.5rem;
-  background-color: rgba(103, 175, 204, 95%);
-  
+  color: white;
+  background-color: rgba(28, 87, 126, 0.8);
+}
+
+#interests {
+  padding-bottom: 0.5rem;
 }
 
 #biography {
   padding-bottom: 2rem;
 }
 
+#tech-stack {
+  background: linear-gradient(to bottom, rgba(28, 87, 126, 0.8), rgba(207, 236, 255, 0.8));
```

Comment:
Nice effect!

Reply:
yes - without this effect the vs-code logo was not good appear